index.css

```diff
@@ -1,635 +0,0 @@
-/* Fade in Animation for page load */
```

Comment:
Nice use of animations throughout your remock! They look very nice when navigating and scrolling through your site.